sdk/messaging_servicebus/src/service_bus/client.rs

```diff
@@ -0,0 +1,132 @@
+use std::sync::Arc;
+
+use crate::service_bus::{
+    peek_lock_message, peek_lock_message2, receive_and_delete_message, send_message,
+    PeekLockResponse,
+};
+use chrono::Duration;
+use ring::hmac::Key;
+
+use azure_core::{
+    error::{Error, ErrorKind, ResultExt},
+    HttpClient,
+};
+
+/// Client object that allows interaction with the ServiceBus API
+pub struct Client {
+    http_client: Arc<dyn HttpClient>,
```

Comment:
This is fine, but do note that we will want to move this crate to using the pipeline architecture (as can be seen [here](https://github.com/Azure/azure-sdk-for-rust/blob/a76ebf194eddf0a1846ffec77078359a5735f5a4/sdk/data_cosmos/src/clients/cosmos.rs#L69-L76)) ASAP. However, we don't need to block this PR on that.

Reply:
Sounds good~ I'll leave this for now, so we can at least get a working `messaging_servicebus` crate out!